frontend/src/components/ThankYouCard/ThankYouCard.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import Checkmark from '../../assets/checkmark-icon.png';
```

Comment:
Did this come from somewhere?

Reply:
Yes. It is an icon from the assets folder that came from icons8. I find that when I name the image as "Checkmark" or "Chicken" or whatever, it allows me to treat it as a component so I don't have to use the "require" to render it.